tornado/web.py

```diff
@@ -399,7 +399,7 @@ def clear_header(self, name: str) -> None:
         if name in self._headers:
             del self._headers[name]
 
-    _INVALID_HEADER_CHAR_RE = re.compile(r"[\x00-\x1f]")
+    _INVALID_HEADER_CHAR_RE = re.compile(r"[\x00-\x08\x0a-\x1f]")
```

Comment:
While we're here: I went back to the RFCs to make sure that this change is correct, and found that `\x7f` (`DEL`) is also invalid. So let's add that to the list too: `\x00-\x08\x0a-\x1f\x7f`. Or for a slightly larger code change we could invert it to define valid chars (like the RFCs do) instead of invalid ones: `\x09\x20-\x7e\x80-0xff`.